h/views/groups.py

```diff
@@ -122,6 +122,11 @@ def read(group, request):
         request.override_renderer = 'h:templates/groups/join.html.jinja2'
         return {'group': group}
 
+    # Redirect to new group overview page if search page is enabled
+    if request.feature('search_page'):
+        url = request.route_path('activity.group_search', pubid=group.pubid)
+        return HTTPSeeOther(url)
```

Comment:
Hmm, I thought you'd need to `raise` these exceptions in Pyramid, but it looks like we always return `HTTPSeeOther` exceptions.